.github/workflows/insitu.yml

```diff
@@ -0,0 +1,31 @@
+name: 🐧 In Situ Vis
+
+on: [push, pull_request]
+
+concurrency:
+  group: ${{ github.head_ref }}-sensei
+  cancel-in-progress: true
+
+jobs:
+  sensei:
+    name: SENSEI
+    runs-on: ubuntu-20.04
+    env:
+      CXX: clang++
+      CC: clang
+      CXXFLAGS: "-Werror -Wshadow -Woverloaded-virtual -Wunreachable-code -fno-operator-names"
+      CMAKE_GENERATOR: Ninja
+      CMAKE_PREFIX_PATH: /root/install/sensei/develop/lib/cmake
+    container:
+      image: ryankrattiger/sensei:fedora33-vtk-mpi-20210616
+    steps:
+    - uses: actions/checkout@v2
+    - name: Configure
+      run: |
+        cmake -S . -B build     \
+          -DWarpX_SENSEI=ON     \
+          -DWarpX_COMPUTE=NOACC \
+          -DCMAKE_CXX_STANDARD=14
```

Comment:
@c-wetterer-nelson @kwryankrattiger both here and in AMReX we need this flag to compile.
I think that's an upstream issue in AMReX that we should fix, we use `target_compile_features(... PUBLIC cxx_std_14)` for both the upstream `amrex` target as well as WarpX and thus this should not be needed, ideally.

I think there might be a requirement missing of the same kind in AMReX that should be fixed:
https://github.com/AMReX-Codes/amrex/blob/989d98b849ebaf31f000f60ff48ffd09304ff621/.github/workflows/linux.yml#L128

Reply:
X-ref upstream: https://github.com/AMReX-Codes/amrex/pull/2115#discussion_r691544975